vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -142,3 +142,58 @@ def categorical_column(data_frame: pd.DataFrame, x: str, y: str):
     # So ticks are aligned with bars when xaxes values are numbers (e.g. years)
     fig.update_xaxes(type="category")
     return fig
+
+
+def offset_signal(signal: float, marker_offset: float):
+    """Offsets a signal value by marker_offset.
+
+    Reduces for positive signal values and increasing for negative values. Used to reduce the length of
+    lines on lollipop charts to end just before the dot.
+
+    Args:
+        signal (float): the value to be updated.
+        marker_offset (float): the offset to be added/subtracted.
+
+    Returns:
+        float: the updated value.
+
+    """
+    if abs(signal) <= marker_offset:
+        return 0
+    return signal - marker_offset if signal > 0 else signal + marker_offset
+
+
+@capture("graph")
+def lollipop(data_frame: pd.DataFrame, x: str, y: str, y_offset: float):
+    """Creates a lollipop chart using Plotly.
+
+    This function generates a scatter chart and then draws lines extending from each point to the x-axis.
+
+    Args:
+        data_frame (pd.DataFrame): The data source for the chart.
+        x (str): The column name to be used for the x-axis.
+        y (str): The column name to be used for the y-axis.
+        y_offset (float): The amount to offset the end of each line from each scatter point.
+
+    Returns:
+        go.Figure: : A Plotly Figure object representing the lollipop chart.
+
+    """
+    data = px.scatter(data_frame, x=x, y=y)
+
+    shapes = []
+    for i, row in data_frame.iterrows():
+        shapes.append(
+            {
+                "type": "line",
+                "xref": "x",
+                "yref": "y",
+                "x0": row[x],
+                "y0": 0,
+                "x1": row[x],
+                "y1": offset_signal(row[y], y_offset),
+                "line": {"color": "grey", "width": 2},
+            }
+        )
+    fig = data.update_layout(shapes=shapes)
```

Comment:
From a style point of view I would probably make the blobs a bit bigger and maybe keep the same blue colour but I'm not sure - I'll leave it to our style guru @huong-li-nguyen to comment.

Reply:
Agree with @antonymilne  👍 I'll propose something to start with